release/__init__.py

```diff
@@ -310,21 +320,33 @@ def add_package(package_id):
     # Add the <variant>.bootstrap.latest as a channel_path
     for name, info in sorted(all_completes.items(), key=lambda kv: pkgpanda.util.variant_str(kv[0])):
         bootstrap_filename = "{}.bootstrap.tar.xz".format(info['bootstrap'])
+        active_filename = "{}.active.json".format(info['bootstrap'])
+        active_local_path = 'packages/cache/bootstrap/' + active_filename
+        latest_filename = "{}bootstrap.latest".format(pkgpanda.util.variant_prefix(name))
```

Comment:
nit: could pull the variant_prefix calculation out and just do it once, then stash the result.